### ISSUE TITLE
Counter-intuitive behavior for input component spellcheck attribute

### ISSUE THREAD

**Issue by krukid** [emberjs/ember.js#18758]
Ember version: 3.16.3

## Expected behavior

```hbs
<Input
  spellcheck="false"
  foobar="false"
  autocomplete="off"
  autocorrect="off"
/>

<!-- RESULTS IN -->
<input
  spellcheck="false"
  foobar="false"
  autocomplete="off"
  autocorrect="off"
>
```

## Actual behavior

```hbs
<!-- RESULTS IN -->
<input
  spellcheck="true" <!-- which is not "false" -->
  foobar="false"
  autocomplete="off"
  autocorrect="off"
>
```

## Live Demos

https://ember-twiddle.com/03f5193ed0724c28650ad575302c6307?openFiles=templates.application%5C.hbs%2C -- input component with curly syntax

https://ember-twiddle.com/14100e6dfaabb6bab22427e11c957004?openFiles=templates.application%5C.hbs%2C -- input element with an event hook

Unfortunately Twiddle/JSBin/Codepen/etc do not support Ember > 3.4, so no live demo with angle syntax :(

## Workarounds

```hbs
<Input @spellcheck={{false}} />
<Input spellcheck={{false}} />
```

I haven't been able to find anything regarding some special handling of this specific attribute in the docs/source.

**Comment by krainboltgreene** [comment]
It's not just angle brackets, this also happens with old school:

``` handlebars
{{input spellcheck="false"}}
```

results in:

``` handlebars
<input spellcheck="true" />
```

**Comment by krukid** [comment]
> It's not just angle brackets, this also happens with old school:
> 
> ```
> {{input spellcheck="false"}}
> ```
> 
> results in:
> 
> ```
> <input spellcheck="true" />
> ```

Missed that :) Added demo, changed title. Thanks!

**Comment by krainboltgreene** [comment]
I also found https://ember-twiddle.com/14100e6dfaabb6bab22427e11c957004?openFiles=templates.application%5C.hbs%2C

**Comment by krukid** [comment]
@krainboltgreene Weird. But also cool.

**Comment by locks** [comment]
Fixed as of 3.17!
